### ISSUE TITLE
cucumer-expressions fails on multiple installs

### ISSUE THREAD

**Issue by yasserf** [cucumber/cucumber-js#1238]
Upgrading from 5 to 6 resulted in npm install failing after the first run

```
npm ERR! path /node_modules/cucumber-expressions
npm ERR! git /node_modules/cucumber-expressions: Appears to be a git repo or submodule.
npm ERR! git     /node_modules/cucumber-expressions
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

**Comment by yasserf** [comment]
You can replicate by

```
git clone git@github.com:deepstreamIO/deepstream.io.git
cd deepstream.io
npm i
npm i // fails
```

**Comment by charlierudolph** [comment]
Hmm interesting. Yep I'm seeing the ".git" folder in that install. Oddly yarn does not the error. Will open an issue on cucumber-expressions

**Comment by aslakhellesoy** [comment]
I'm not sure how this happened! Until we've fixed this, adding `"cucumber-expressions": "7.0.1"` to `package.json` seems to work.

As for a more permanent fix we can either release `cucumber-expressions 7.0.3` (and make sure that `.git` folder is excluded), or we could make a patch-release of `cucumber` where we upgrade to `cucumber-expressions 8.x`.

The latter would be my preference, since the `7.x` release process had several issues that have been completely overhauled in `8.x`. The only thing holding up upgrading to `8.x` is https://github.com/cucumber/cucumber/issues/725. Once 8.0.1 is out with a fix for that we can make another release of cucumber.

WDYT @charlierudolph?

**Comment by charlierudolph** [comment]
Thanks making a release with this now
